packages/patternfly-4/react-core/src/components/Dropdown/DropdownToggle.js

```diff
@@ -5,10 +5,10 @@ import Toggle from './Toggle';
 import styles from '@patternfly/patternfly-next/components/Dropdown/dropdown.css';
 import { css } from '@patternfly/react-styles';
 
-const DropdownToggle = ({ children, ...props }) => (
+const DropdownToggle = ({ children, noCaret, ...props }) => (
   <Toggle {...props}>
     {children}
-    <CaretDownIcon className={css(styles.dropdownToggleIcon)} />
+    {!noCaret && <CaretDownIcon className={css(styles.dropdownToggleIcon)} />}
```

Comment:
As an alternative API, what about adding an `IconComponent` prop that can be set to null and defaults to the `CaretDownIcon`? This would allow it to be replaced (maybe with a kebab icon) or removed by setting it to null.

Reply:
nice, I like that idea as well, going to give it a whirl

Reply:
hmm interesting. I don't know if PF design will allow different caret icons there but I could see the case for this. This meets the immediate need I am having though (but could also be met by providing a null icon). I think `noCaret` is a bit more explicit right now. I am happy either way but am fine w/ the approach here for now.